### PR TITLE
feat(route): add 通信人家园

### LINKED ISSUE #9210: 114通信家园 最新主题
### 类型

论坛

### 网站地址

https://www.txrjy.com/c114-listnewtopic.php

### 网站描述

通信人家园

### 需要生成什么内容？

生成RSS订阅源

### 额外描述

_No response_

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by Fatpandac**
## 该 PR 相关 Issue / Involved issue

Close #9210

## 完整路由地址 / Example for the proposed route(s)

```routes
/txrjy/fornumtopic/
/txrjy/fornumtopic/2
/txrjy/fornumtopic/3
/txrjy/fornumtopic/4
/txrjy/fornumtopic/5
/txrjy/fornumtopic/6
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [X] 新的路由 New Route
  - [X] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [X] 文档说明 Documentation
  - [X] 中文文档 CN
  - [ ] 英文文档 EN
- [X] 全文获取 fulltext
  - [X] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [X] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [X] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: docs/bbs.md [hunk 1]
```diff
@@ -654,6 +654,18 @@ pageClass: routes
 
 </Route>
 
+## 通信人家园
+
+### 论坛 频道
+
+<Route author="Fatpandac" example="/txrjy/fornumtopic" path="/txrjy/fornumtopic/:chennel?" :paramsDesc="['频道的 id，见下表，默认为最新500个主题帖']">
```
Comment (TonyRL) [inline docs/bbs.md:661]:
```suggestion
<Route author="Fatpandac" example="/txrjy/fornumtopic" path="/txrjy/fornumtopic/:channel?" :paramsDesc="['频道的 id，见下表，默认为最新500个主题帖']">
```

File: lib/v2/txrjy/fornumtopic.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+const iconv = require('iconv-lite');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+const rootUrl = 'https://www.txrjy.com';
+
+module.exports = async (ctx) => {
+    const chennel = ctx.params.channel ?? '1';
+    const url = `${rootUrl}/c114-listnewtopic.php?typeid=${chennel}`;
```
Comment (TonyRL) [inline lib/v2/txrjy/fornumtopic.js:12]:
```suggestion
    const channel = ctx.params.channel ?? '1';
    const url = `${rootUrl}/c114-listnewtopic.php?typeid=${channel}`;
```

**Review by TonyRL — CHANGES_REQUESTED** [review]

File: lib/v2/txrjy/templates/fornumtopic.art [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<h4>{{ author }}</h4>
```
Comment (TonyRL) [inline lib/v2/txrjy/templates/fornumtopic.art:1]: Do not put author in description

Comment (Fatpandac) [reply lib/v2/txrjy/templates/fornumtopic.art:1]: The author is used to mark who commented on this post,
So I think it would be better to keep it.

Comment (TonyRL) [reply lib/v2/txrjy/templates/fornumtopic.art:1]: :ok_hand: saw it

**Review by TonyRL — CHANGES_REQUESTED** [review]
`const timezone = require('@/utils/timezone')`

**Pull request merged**